packages/docs/blog/2021-07-09-remotion-2-2.md

```diff
@@ -0,0 +1,165 @@
+---
+slug: 2-2
+title: Remotion 2.2
+author: Jonny Burger
+author_title: Remotion Creator
+author_url: https://github.com/JonnyBurger
+author_image_url: https://avatars2.githubusercontent.com/u/1629785?s=460&u=12eb94da6070d00fc924761ce06e3a428d01b7e9&v=4
+---
+
+import { ThreeDPlayer } from "../components/3DPhonePlayer.tsx";
+
+Welcome to the release notes of Remotion 2.2! It's been a while since v2.1, but in the meanwhile we had a dozen smaller releases inbetween. This post summarizes the highlights of the past 2 months 🎉.
+
+## Environment variable support
+
+Us developers use environment variables extensively to manage configuration, secrets and other stuff you don't want to commit to a Github repo. If you write videos with code, environment variables are useful too!
+
+We added support for environment variables from the CLI, using a `.env` file, and allowing you to pass variables via our Node.JS APIs too. [Click here to learn more.](/docs/env-variables)
+
+## `<Freeze />` component
+
+This new core component will freeze all of it's children and make them think that the video is paused at a certain time.
+
+Use it for example to display a still frame from a video:
+
+```tsx twoslash title="MyVideo.tsx"
+import {Freeze, Video} from 'remotion';
+import myVid from './vid.mp4';
+
+export const MyVideo = () => {
+  return (
+    <Freeze frame={30}>
+      <Video src={myVid} />
+    </Freeze>
+  )
+}
+```
+
+However, it works for any content. As you can see the API is dead simple. You can combine it with the [`<Sequence>`](/docs/sequence) API to make any content play, pause, and then continue again.
```

Comment:
... any content play, pause and then continue again. (I'm not sure but maybe without a comma after "pause")